be/app/Http/Controllers/API/UserController.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace App\Http\Controllers\API;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use App\User;
+
+class UserController extends Controller
+{
+ /*  /*   *
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        return User::all();
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        $user = User::create($requests->all());
+        return response()->json($user, 200);
+    }
```

Comment:
- please remove "API" directory
![image](https://user-images.githubusercontent.com/73863685/218981612-0fbb8f0f-d2f5-4eac-9bef-1629288777fa.png)

- I'm confused, which controller jud imo ginagamit pag register. This controller or the "AuthController", please remove ko sa osa ka controller.